packages/react/src/components/Message/Message.js

```diff
@@ -235,6 +235,7 @@ const Message = ({
           {!message.t && showToolbox ? (
             <MessageToolbox
               message={message}
+              RCInstance={RCInstance}
```

Comment:
Why is this added ?

Reply:
Hey @Spiral-Memory the adding of RCInstance as a prop was unnecessary, after removing it I have seen no change